views/panel.php

```diff
@@ -57,7 +57,7 @@
   </script>
 
   <?php foreach ($assets['js'] as $js): ?>
-  <script type="module" nonce="<?= $nonce ?>" src="<?= $js ?>"></script>
+  <?= Html::tag('script', '', $js) ?>
```

Comment:
I'm not sure but currently all assets in one line. To break line, need to append `. "\n"`.

Reply:
Sounds good to me

Reply:
Which one? Is it a one line or adding a break line? 😅

Reply:
The line break